static/js/urlparser.js

```diff
@@ -1,9 +1,15 @@
-function getUrlParam(key){
-    var urlParams = new URLSearchParams(window.location.search);
-    var param = urlParams.get(key);
-    if(param === null){
-        console.log("that url parameter key does not exist");
-        return;
-    }
-    return param;
-}
\ No newline at end of file
+function getUrlParam(key) {
+    var urlParams = window.location.search.split("?")[1];
+    var value = null;
+    urlParams = urlParams.split("&");
+    urlParams.forEach(function (param) {
+        let parameter = {
+            key: param.split("=")[0],
+            value: param.split("=")[1]
+        }
+        if (parameter.key === key) {
+            value = parameter.value;
+        }
+    });
+    return value;
+};
```

Comment:
I would recommend to write some tests for different URL formats:
- `.org/v/1`
- `.org/v/1#anchor`
- `.org/v/1?key=value`
- `.org/v/1?key1=value1&key2=value2`
- `.org/v/1?key1=value1&key2=value2#anchor`
- `.org/v/1?key1&key2#anchor`
- `.org/v/1?key=`
- `.org/v/1?key=value&`
- `.org/v/1?key=&`
- `.org/v/1?key=&#anchor`

You can create a new tests/spec/urlparser-spec.js and use Jest.